client/blocks/app-banner/utils.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Internal dependencies
+ */
+import {
+	get,
+	includes,
+	reduce,
+} from 'lodash';
+
+export const PREFERENCE_NAME = 'appBannerDismissTimes';
```

Comment:
Sure, that sounds better. Refactored in 0231c21a5cc3173a627ef787cd2ac08a24712505.